src/scrapers/union-bank.ts

```diff
@@ -274,10 +274,7 @@ async function fetchAccounts(page: Page, startDate: Moment) {
 }
 
 async function waitForPostLogin(page: Page) {
-  return Promise.race([
-    waitUntilElementFound(page, '#signoff', true),
-    waitUntilElementFound(page, '#restore', true),
-  ]);
+  return waitUntilElementFound(page, '.signoff_button', false);
```

Comment:
@orentet why are we changing this logic? doesn't it work as it is?

Reply:
no. the UI was probably changed.